packages/babel-plugin-lingui-macro/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@lingui/babel-plugin-lingui-macro",
+  "version": "4.8.0-next.1",
+  "description": "Babel plugin for transforming Lingui Macros",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "maintainers": [
```

Comment:
https://github.com/npm/www/issues/133#issuecomment-284906561

it should be `contributors` in this case

Reply:
> We don't use such a field in any package. 

We should start. This will be first one.

Reply:
I don't see any reason why it would be needed at all. 

> it should be contributors in this case

This will require additional maintenance of the list which is not needed work.

There is a [contributors](https://github.com/lingui/js-lingui/graphs/contributors) section in the repository, and that's sufficient.

Reply:
I want to have my name written, there. Only key contributors/maintainers of the package should be here.

Reply:
Ok, I'm okay with the `contributors`